packages/app/cypress/e2e/runner/ui-states.cy.ts

```diff
@@ -0,0 +1,260 @@
+import { runSpec } from './support/spec-loader'
+
+describe('src/cypress/runner ui states', { retries: 0, defaultCommandTimeout: 600000 }, () => {
+  beforeEach(() => {
+    cy.viewport(1000, 1500)
+  })
+
+  it('hooks', () => {
+    runSpec({
```

Comment:
instead of `runSpec`, can we leverage `loadSpec` with the correct num of 'pass`/`fail` ed assertions to avoid the `cypress:in:cypress:run:complete` event? That method _should_ wait for the test to finnish